### DISCUSSION TITLE
Impressões

### DISCUSSION THREAD

**Discussion by almeida-pf** [RamonSilva20/mapos#1772] [General]
Olá, é possível imprimir algum documento fiscal pelo sistema? e caso queira imprimir em uma impressora não fiscal o documento já esta adaptado precisa fazer alguma alteração no modelo?

**Comment by Pr3d4dor** [comment]
@almeida-pf Impressão não fiscal já tem, impressão fiscal não tem pois não é tão simples implementar.

**Reply by almeida-pf** [reply]
Entendi, e tem como configurar um modelo na impressão? por exemplo adicionar informações colocar algum campo especifico.
